yjit_codegen.c

```diff
@@ -657,43 +677,45 @@ yjit_gen_block(block_t *block, rb_execution_context_t *ec)
 
         // Lookup the codegen function for this instruction
         codegen_fn gen_fn = gen_fns[opcode];
-        if (!gen_fn) {
```

Comment:
This change unifies the `!gen_fn` case and the case where `gen_fn` returns `CANT_COMPILE`. They were doing the same thing